packages/plugin-console-breadcrumbs/console-breadcrumbs.js

```diff
@@ -1,14 +1,12 @@
-const { map, reduce, filter } = require('@bugsnag/core/lib/es-utils')
+const { map, reduce, filter, includes } = require('@bugsnag/core/lib/es-utils')
 
 /*
  * Leaves breadcrumbs when console log methods are called
  */
 exports.init = (client) => {
   const isDev = /^dev(elopment)?$/.test(client.config.releaseStage)
 
-  const explicitlyDisabled = client.config.consoleBreadcrumbsEnabled === false
-  const implicitlyDisabled = (client.config.autoBreadcrumbs === false || isDev) && client.config.consoleBreadcrumbsEnabled !== true
-  if (explicitlyDisabled || implicitlyDisabled) return
+  if (!client.config.enabledBreadcrumbTypes || !includes(client.config.enabledBreadcrumbTypes, 'log') || isDev) return
```

Comment:
This change has the known impact of meaning console log breadcrumbs can't be enabled in development.